.devcontainer/Dockerfile

```diff
@@ -44,5 +44,6 @@ RUN git clone --branch ${FLUTTER_CHANNEL} https://github.com/flutter/flutter.git
 	&& flutter config --no-analytics \
 	&& dart --disable-analytics
 
-# Activate melos
-RUN dart pub global activate melos
+# Activate aft
+# TODO(Jordan-Nelson): enable aft in containers
```

Comment:
What's blocking this?

Reply:
It was failing to activate from path. I think the dir I have here is wrong, but debugging these are a pain (at least I am not aware of a good way to do it). I think ideally it would run aft bootstrap, but it isn't as necessary as it was when we were using melos since we can do a typical pub get now.

Reply:
Ah it's because the workspace is not mounted when building the Docker container, only when running it afterwards. You can set it in `devcontainer.json` as a `postStartCommand`:

```json5
{
  // ...
  "postStartCommand": "dart pub global activate -spath packages/aft"
}

Reply:
That makes sense. Updated.